os_dep/linux/ioctl_cfg80211.c

```diff
@@ -4570,6 +4570,42 @@ static int	cfg80211_rtw_set_channel(struct wiphy *wiphy
 	return 0;
 }
 
+#if (LINUX_VERSION_CODE >= KERNEL_VERSION(3, 8, 0))
+static int	cfg80211_rtw_set_monitor_channel(struct wiphy *wiphy, struct cfg80211_chan_def *chandef)
+{
+    struct ieee80211_channel *chan = (struct ieee80211_channel *) chandef->chan;
```

Comment:
Could you replace the spaces with tabs here?
(sorry, I'm a sucker for code consistency)